.github/workflows/api.yml

```
@@ -29,13 +31,13 @@ jobs:
       - name: Log in to GitHub Docker Registry
         uses: docker/login-action@v1
         with:
-          registry: ${{ secrets.REGISTRY }}
-          username: ${{ secrets.CR_USER }}
-          password: ${{ secrets.CR_PWD }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.ARGON_GHCR_PAT }}
       - name: Build and push
         uses: docker/build-push-action@v2
```

actionlint

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)